modules/rostests/apitests/ntdll/NtApphelpCacheControl.c

```diff
@@ -363,5 +373,3 @@ START_TEST(NtApphelpCacheControl)
         CloseHandle(service_stop_event);
     }
 }
-
-
```

Comment:
Why?

Reply:
Multiple end of file newlines are not that useful, are they? (Though I care less about removing them.)

Reply:
Make up your mind. Is thia about whitespace changes or making the text run in xp (which is not really worth the work already put in it).